index.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Portofolio</title>
```

Comment:
It will be better and more descriptive if your title says more about your page, a title like " Damena Bootstrap Website Portfolio would make more sense or what do you think?